packages/web3-eth-accounts/src/account.ts

```diff
@@ -142,17 +135,16 @@ export const signTransaction = (
 		throw new SignerError(errorString);
 	}
 
-	const rlpEncoded = signedTx.serialize().toString('hex');
```

Comment:
why you are skipping rlp encoding tx?

Reply:
It's not skipped, just wrapped in `bytesToHex`

Reply:
There was a bug in this function, which was causing different transaction hash calculation compared to one if you send raw transaction to network you get different tx hash. I fixed it.